### PR TITLE
Forest of Prayer

### PR DESCRIPTION
**Pull request by vtange**
That's a lot of Sanctums

### CODE REVIEW

**Comment by Sunaries** [comment]
:+1:

File: QUEST_LV_0200.tsv [hunk 1]
```diff
@@ -2809,30 +2809,30 @@ QUEST_LV_0200_20150317_002808	Spray the monster fluid
...
-QUEST_LV_0200_20150317_002820	{memo X}Offered enough sacrifice. Worship sanctuary to receive blessings.
-QUEST_LV_0200_20150317_002821	Offer worship to sanctuary
-QUEST_LV_0200_20150317_002822	{memo X}Gathered monster meat to use as sacrifice and offered worship but it was a polluted sanctuary!
-QUEST_LV_0200_20150317_002823	Defeat the monsters near the sanctuary
-QUEST_LV_0200_20150317_002824	{memo X}Monsters swarmed out when you prayed to the sanctuary! Looks like the monsters appeared after smelling the meat because the sanctuary is polluted. Defeat the monsters first then look for a way to purify the sanctuary.
-QUEST_LV_0200_20150317_002825	Go to the Priest Master
-QUEST_LV_0200_20150317_002826	{memo X}The Priest Master manages this sanctuary so he must know how to purify it. It's going to be a long journey, but let's begin.
-QUEST_LV_0200_20150317_002827	Defeat the monsters that gathered around the sanctuary
+QUEST_LV_0200_20150317_002818	{memo X}If you want to worship the Sanctum properly, you need to offer sacrifice as written in the signboard. Lure the monsters nearby to the Sanctum and defeat it.
+QUEST_LV_0200_20150317_002819	$Pay Worship the Sanctum
```
Comment (terrapin47) [inline QUEST_LV_0200.tsv:2819]: to the

File: QUEST_LV_0200.tsv [hunk 2]
```diff
@@ -2809,30 +2809,30 @@ QUEST_LV_0200_20150317_002808	Spray the monster fluid
...
+QUEST_LV_0200_20150317_002820	{memo X}Offered enough sacrifice. Worship Sanctum to receive blessings.
+QUEST_LV_0200_20150317_002821	$Offer worship to Sanctum
```
Comment (terrapin47) [inline QUEST_LV_0200.tsv:2821]: to the

File: QUEST_LV_0200.tsv [hunk 3]
```diff
@@ -3253,40 +3253,40 @@ QUEST_LV_0200_20150323_003252	You may have to dedicate some flowers, or you may
...
+QUEST_LV_0200_20150323_003282	$By 'corrupted Sanctums', you mean the Forest of Prayer, right?{nl}I thought I had prepared a lot for that, but I guess Naktis' curse was even stronger.
+QUEST_LV_0200_20150323_003283	$I hid my Symbol of Faith in the Sanctums to prepare for times like this.{nl}That symbol enables you to find the monsters who have swallowed the Badges of the Sanctum.{nl}
+QUEST_LV_0200_20150323_003284	$Find the Badges of the Sanctum and burn them in front of the Sanctum.{nl}It may not return to how it used to be, but you will be able to purify it a bit.
+QUEST_LV_0200_20150323_003285	$The blessing that cleanses evil energy.{nl}Oh Goddesses, bless us with purified water.
+QUEST_LV_0200_20150323_003286	$A floral tribute to the Goddesses.{nl}The pleasure and joy of collecting flowers.
+QUEST_LV_0200_20150323_003287	$I was able to get one step closer to the Goddesses due to my contribution.{nl}May the blessing be with you always.
+QUEST_LV_0200_20150323_003288	$You know that Pilgrim's Way holds the lessons of Maven right?{nl}Naktis is so formidable. She was able to cast a curse on humans as if they originally sinned.
```
Comment (terrapin47) [inline QUEST_LV_0200.tsv:3288]: Isn't "amazing" correct?  She's talking about the way in which Naktis cast the curse.

Comment (vtange) [inline QUEST_LV_0200.tsv:3288]: It makes it sound like the Pardoner Master is praising Naktis. Prefer a word that merely respects Naktis' potential and ability

**Comment by imcgames** [comment]
:+1:

File: QUEST_LV_0200.tsv (hunk 3 above)
Comment (terrapin47) [inline QUEST_LV_0200.tsv:3288]: True, but formidable is kind of like respect through strength/force/power.  This is more like an interesting irony that Naktis was able to pull off.

**Pull request merged**